instrumentation/struts/struts-2.3/javaagent/build.gradle.kts

```diff
@@ -27,7 +27,7 @@ dependencies {
   testInstrumentation(project(":instrumentation:jetty:jetty-8.0:javaagent"))
   testInstrumentation(project(":instrumentation:struts:struts-7.0:javaagent"))
 
-  latestDepTestLibrary("org.apache.struts:struts2-core:6.0.+")
+  latestDepTestLibrary("org.apache.struts:struts2-core:6.0.+") // see struts2-7.0 module
```

Comment:
Latest version of struts 6.x is `6.7.0` which we don't test. Noticed this in the struts 7 PR and thought about investigating, but immediately forgot about it.